kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx

```diff
@@ -34,4 +34,51 @@ describe("Deployment > Deployment history tests", () => {
 
     expect(getByText("No deploy history")).toBeInTheDocument();
   });
+
+  it('should render deploy history', () => {
+    vi.spyOn(hooks, 'useGetAPIDeployments').mockReturnValue({
+      data: {
+        data: [
+          {
+            "targetMapperKey": "mef.sonata.api-target-mapper.order.eline.read.delete",
+            "path": "/mefApi/sonata/productOrderingManagement/v10/productOrder/{id}",
+            "method": "get",
+            "requiredMapping": true,
+            "diffWithStage": true,
+            "productType": "access_e_line",
+            "actionType": "delete",
+            "mappingMatrix": {
+              "productType": "access_e_line",
+              "actionType": "delete"
+            },
+            "componentName": "Product Ordering Management",
+            "componentKey": "mef.sonata.api.order",
+            "envId": "7da9098b-107f-402c-9d6d-20b1be96b99a",
+            "envName": "production",
+            "createAt": "2024-11-18T10:06:14.412978Z",
+            "createBy": "24044233-3683-425e-9cbe-62e643f081d8",
```

Comment:
please change the testing data, do not expose personal name.

Reply:
Also please make sure there is no code smell in pr.

Reply:
I copied api response and forgot to change it. Won't producing same issue again.